packages/manager/apps/dedicated/client/app/account/contacts/update/index.js

```diff
@@ -0,0 +1,41 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'ovhManagerDedicatedAccountContactsUpdateLazyLoading';
+
+angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider.state('app.account.contact-update-domain.**', {
+      url: '/contact/:currentDomain/:contactId',
+      lazyLoad: ($transition$) => {
+        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+        return import(
+          /* webpackChunkName: "contact-update" */ './user-contacts-update.module'
+        ).then((mod) => $ocLazyLoad.inject(mod.default || mod));
+      },
+    });
+
+    $stateProvider.state('app.account.contact-update.**', {
+      url: '/contact/:contactId/',
+      lazyLoad: ($transition$) => {
+        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+        return import(
+          /* webpackChunkName: "contact-update" */ './user-contacts-update.module'
+        ).then((mod) => $ocLazyLoad.inject(mod.default || mod));
+      },
+    });
+
+    $stateProvider.state('app.account.contact-update2.**', {
+      url: '/contact/:contactId',
+      lazyLoad: ($transition$) => {
+        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+        return import(
+          /* webpackChunkName: "contact-update" */ './user-contacts-update.module'
+        ).then((mod) => $ocLazyLoad.inject(mod.default || mod));
+      },
+    });
```

Comment:
this looks useless right ?

Reply:
I'm not sure as there is no trailing slash for this one and the name of the state is not the same. Further testing should be done and maybe remove this in a separate PR if indeed not useful if it sounds good to you